### PR TITLE
Add message-broker to development.sh

### PR DESCRIPTION
**Pull request by gagansh7171**

### CODE REVIEW

File: scripts/start/development.sh [hunk 1]
```diff
@@ -4,4 +4,5 @@
 docker-compose up -d \
     database \
     redis-gui \
-    cache
\ No newline at end of file
+    cache\
```
Comment (dhruvkb) [inline scripts/start/development.sh:7]:
```suggestion
    cache \
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
LGTM except for a minor change.

**Comment by gagansh7171** [comment]
Committed requested change.

**Review by dhruvkb — APPROVED** [review]

**Comment by algomaster99** [comment]
@gagansh7171 good work with writing commit messages. :smile:

**Pull request merged**